src/components/ui/Tag/index.js

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React, { forwardRef } from 'react';
+
+import { Icon } from 'components/ui/Icon';
+import { getContrastingTextColor, metadata, toUnits } from 'styles';
+import { useTheme } from 'themes';
+
+export const Tag = forwardRef((props, ref) => {
+  const theme = useTheme();
+  const { children, label, onDismiss, ...otherProps } = props;
+
+  return (
+    <span ref={ref} {...otherProps}>
```

Comment:
This is why you're getting the `count` error; you're passing `otherProps`, which includes `otherProps.count` in here to a `<span>`, which doesn't support the `count` prop. You need to "extract" it above:

```jsx
  const { children, count, label, onDismiss, ...otherProps } = props;
```

Reply:
Ohhhh I didn't think I needed to do that, since `count` isn't being used by this function. Does that mean I need to pull in `status` as well?

Reply:
Correct! It's a bit awkward because, as you say: it isn't being used. But yeah, it's how those object spreads work. 🤷‍♂

Reply:
Why is it that `status` doesn't generate an error then? (Or `colour`, for that matter, or other props used similarly in other components?) Is this something we need to be checking for?

Reply:
Because they're `undefined` by default and ES6 spreads omit keys with a value of `undefined` through. But because this is `false` by default it gets passed through.

We're already checking for it and it's working. 😄